lib/api/core/storage/indexEngine.js

```diff
@@ -73,7 +116,7 @@ class InternalStorage {
    *
    * @returns {Promise.<Object>} { scrollId, hits, aggregations, total }
    */
```

Comment:
The `IndexEngine` methods has to keep the same signature because they are used by [plugin Repository API](https://docs.kuzzle.io/core/1/plugins/plugin-context/constructors/repository/)